java/demo/pom.xml

```diff
@@ -1,67 +1,21 @@
 <?xml version="1.0"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <groupId>com.googlecode.libphonenumber</groupId>
+  <artifactId>demo</artifactId>
+  <version>8.3.4-SNAPSHOT</version>
+  <packaging>jar</packaging>
```

Comment:
So adding <packaging> and dropping the <repositories> did the trick?
Thanks for figuring that out!